File: Jour17/jour17.pas
program Min_Max;
var
    Tab : array[1..100] of integer;
    i, n, min, max    : integer;
begin
	writeln('SAISIE DE VOS DONNEES');
	while True do
	begin
        write('Combien d"entier voulez vous fournir ? : ');
        readln(n);
        if ((n < 0) or (n > 100)) then
        begin
        	writeln('Erreur depacement ::: votre choix est entre 1 et 100');
        	continue;
        end
        else
            break;
    end;
    for i := 1 to n do
    begin
    	write('Donner l"element ',i, ': ');
    	read(Tab[i]);
    end;
    min := Tab[1];
    max := Tab[1];
    for i := 1 to n do
    begin
    	if min > Tab[i] then
            min := Tab[i];
        if max < Tab[i] then
            max := Tab[i];
    end;
    writeln('Le minimum de vos donnees vaut : ',min,' et le maximum ',max,'.');
end.

File: Jour17/jour17.c
#include<stdio.h>

int main()
      {
	int Tab[100];
	int i, n, min, max;
	printf("\nSAISIE DES DONNEES.\n");
	do
	{
	       printf("Combien d'entier voulez-vous saisir ? : ");
	       scanf("%d",&n);
	       if(n < 0 || n > 100)
	             printf("Erreur votre choix doit etre entre 1 et 100 Merci.\n");
	}while(n < 0 || n > 100);
	for(i = 0; i < n; i++)
	{
	        printf("Donner l'element %d : ",i+1);
	        scanf("%d",Tab+i);
	}
	min = Tab[0];
	max = Tab[0];
	for(i = 0; i < n; i++)
	{
	        if(min > Tab[i])
	            min = Tab[i];
	        if(max < Tab[i])
	            max = Tab[i];
	}
	printf("Le minimum de vos donnees vaut : %d et le maximum vaut : %d\n",min,max);
	return 0;
       }